configs/components/oifs/oifs.yaml

```diff
@@ -764,19 +773,6 @@ choose_wam:
 # OTHER CONFIGS AND SWITCHES
 #======================================================================================
 
-# For coupled setups we can use ocean currents
-choose_use_ocean_currents:
-        1:
-                add_namelist_changes:
-                        fort.4:
-                                NAEPHY:
-                                        LECURR: ".true."
-        0:
-                add_namelist_changes:
-                        fort.4:
-                                NAEPHY:
-                                        LECURR: ".false."
-
```

Comment:
**Systems involved**: AWI-CM3, FOCI-OpenIFS, OpenIFS standalone (@seb-wahl, @joakimkjellsson)
    - **Evaluation**: no effect on tested simulations since namelists are not affected
    - **Details**: https://github.com/esm-tools/esm_tools/runs/7038392238?check_suite_focus=true#step:6:1299. `LECURR: False` is not in `namelist_changes` anymore. This doesn't seem to affect the `fort.4` namelist for the tests, in any of the systems, since the default is False. However, bare in mind that the `oifs` switch`use_ocean_currents` will not longer work.
    - **Action required**: @seb-wahl and @joakimkjellsson please confirm that you are not using the `use_ocean_currents` switch and that it can be deleted.

Reply:
Thanks for double checking. I think the switch whether Ocean current coupling is used should not be in oifs.yaml in the first place, but rather in the respective setup yaml files (foci,yaml, awicm3.yaml, oifsamip.yaml), as this is something that only makes sense in a coupled system. 

I also had trouble overwriting this from the awicm3.yaml. It kept defaulting back onto use_ocean_currents: 0 and thus LECURR: ".false." even when I set use_ocean_currents: 1 inside the choose_version block in oifs.yaml.

Reply:
Hi @mandresm 
Feel free to remove this. We can for sure do the "add_namelist_changes" in the runscript. 

Cheers
Joakim